docs/UserGuide.md

```diff
@@ -92,11 +92,12 @@ All commands are case-sensitive.
 
 * Parameters can be in any order.<br>
   e.g. if the command specifies `-name {NAME} -address {ADDRESS}`, `-address {ADDRESS} -name {NAME}` is also acceptable.
-
+* Prefixes for parameters need to have a space before the `-`. <br>
+e.g. `add -name Xiao Ming -address 13, Computing Dr, 117417 -email xiaoming@email.com -phone 88888888 -subject Math` is valid, but `add-name Xiao Ming-address 13, Computing Dr, 117417-email xiaoming@email.com-phone 88888888-subject Math` is not.
 * All id inputs in commands labelled `{ID}` can omit any leading 0s.<br>
 e.g. `1`, `01`, `000000001` are all valid id inputs for the id `#000001`
 
-* All `{ID}` are by default a positive integer of maximum *6 digits*. Any ID that exceeds a positive 6 digit integer number (e.g. 1234567) may cause errors.
+* All `{ID}` are by default a positive integer of maximum *6 digits* excluding leading 0s. Any ID that exceeds a positive 6 digit integer number (e.g. 1234567) may cause errors. However, 000000001 is still a valid ID as leading 0s are ignored.
```

Comment:
would it be better to add the example of 00000000999999 is valid if student #999999 exists to make the point of positive integer of max 6 digits excluding leading 0s?

Reply:
I think "However, 000000001 is still a valid ID as leading 0s are ignored." will be more suited if moved up to line 97. 

e.g. `1`, `01`, `000000001` are all valid id inputs for the id `#000001` **as leading 0's are ignored.**